test/tests/mqtrigger/kafka/kafka_pub/kafka-pub.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	sarama "github.com/Shopify/sarama"
+)
+
+// Handler posts a message to Kafka Topic
+func Handler(w http.ResponseWriter, r *http.Request) {
+	brokers := []string{"broker.kafka.svc.cluster.local:9092"}
+	producerConfig := sarama.NewConfig()
+	producerConfig.Producer.RequiredAcks = sarama.WaitForAll
+	producerConfig.Producer.Retry.Max = 10
+	producerConfig.Producer.Return.Successes = true
+	producer, err := sarama.NewSyncProducer(brokers, producerConfig)
+	fmt.Println("Created a new producer ", producer)
+	if err != nil {
+		panic(err)
+	}
+	_, _, err = producer.SendMessage(&sarama.ProducerMessage{
+		Topic: "testtopic",
+		Value: sarama.StringEncoder("{\"name\": \"testvalue\"}"),
+	})
+
+	if err != nil {
+		w.Write([]byte(fmt.Sprintf("Failed to publish message to topic %s: %v", "testtopic", err)))
+	}
+	w.Write([]byte("Successfully sent to testtopic"))
```

Comment:
If you haven't, you could use this test also as an example. I am sure it would be useful for users to have an example of Kafka in the examples directory.

Reply:
Agreed, will send a separate PR.